install/docker/index.markdown

```diff
@@ -34,7 +34,7 @@ And you will likely need to log out and back into your user account for the chan
 To run a Debian-installed container of MoveIt! with graphics support:
 
     wget https://raw.githubusercontent.com/ros-planning/moveit/melodic-devel/.docker/gui-docker -O gui-docker && chmod +x gui-docker
-    ./gui-docker -it --rm moveit/moveit:melodic-release /bin/bash
+    ./gui-docker -it --rm moveit/moveit:melodic-release
```

Comment:
removing /bin/bash breaks the interactivity right?

Reply:
It shouldn't due to the default entrypoint, but I suppose with it is more explicit about whats going on and that's probably better. So I'll add it back in.